Yumemi-ios-training/Yumemi-ios-training/Model/WeatherFetcher.swift

```diff
@@ -31,4 +40,17 @@ class WeatherFetcher: WeatherFetchable {
             fatalError("天気情報取得時に予期せぬエラーが発生：\(error.localizedDescription)")
         }
     }
+    
+    func parseJson(with jsonString: String) -> WeatherResponse? {
```

Comment:
[IMO]
```suggestion
    func parseJson(with jsonString: String) throws -> WeatherResponse {
```
`return nil`でなく、`throw jsonDecodeError`できる